challenge-03/qosf-monthly-challenge-03.ipynb

```diff
@@ -0,0 +1,56 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Challenge 03: Minimum Makespan with Quantum Annealing\n",
+    "\n",
+    "Niranjan owns a catering service and receives orders one day in advance. Each of the chefs he employs is payed by the hour. In order to optimize the cost of running his business, every night, he looks at all of the dishes he needs to deliver for the next day and asks his employees to work the specific hours he requires them for.\n",
+    "\n",
+    "Given an order queue of length $N$ with each order $i$ taking $L_i$ time to prepare, if there are $m$ chefs, design a QUBO/Ising Hamiltonian to help Niranjan find the lowest time it would take for the $m$ chefs to prepare all $N$ dishes. \n",
+    "\n",
+    "Since Niranjan wants to ensure that he can run his QUBO/Ising Hamiltonian on his QC providers' free tier for as long as possible, also minimize the number of:\n",
```

Comment:
I would rephrase to:
"Since Niranjan wants to ensure that he can run his algorithms on the near term devices, keep track and try to minimize the number of:
- Gates and Qubit (for Ising Hamiltonian)
- Variables (for QUBO. For embedding on a real machine also keep track of the lenghts of chains).